src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -25,8 +25,11 @@ import { IModelContentChangedEvent } from 'vs/editor/common/model/textModelEvent
 import { firstIndex, find } from 'vs/base/common/arrays';
 import { HideInputTag } from 'sql/platform/notebooks/common/outputRegistry';
 import { FutureInternal, notebookConstants } from 'sql/workbench/services/notebook/browser/interfaces';
+import { ICommandService } from 'vs/platform/commands/common/commands';
+import { tryMatchCellMagic, extractCellMagicCommandPlusArgs } from 'sql/workbench/services/notebook/browser/utils';
 
 let modelId = 0;
+const ADS_EXECUTE_COMMAND = 'ADS_EXECUTE_COMMAND';
```

Comment:
It looks like most existing magics are lower case - any reason you made this upper?